libs/stats/odc/stats/_gm_ls_bitmask.py

```diff
@@ -25,19 +23,32 @@ def __init__(
             self,
             bands: Optional[Tuple[str, ...]] = None,
             mask_band: str = "QA_PIXEL",
+            # provide flags with high cloud bits definition
+            flags: Dict[str, Optional[Any]] = dict(
+                cloud="high_confidence",
+                # cloud_shadow="high_confidence",
```

Comment:
Shouldn't we have `cloud_shadow` masked by default @fangfy ?

Reply:
> Shouldn't we have `cloud_shadow` masked by default @fangfy ?

That was one of the test options I used because I notice a lot of false cloud shadow in the city. However I agree maybe in the end, we will still just mask all cloud_shadow.